cmake/modules/configuration_files.cmake

```diff
@@ -39,7 +39,17 @@ else()
   set(APPLICATION_CONFIG_DIR ${APPLICATION_SOURCE_DIR})
 endif()
 
+zephyr_get(global_conf_file SYSBUILD GLOBAL VAR CONF_FILE)
 zephyr_get(CONF_FILE SYSBUILD LOCAL)
+
+if(DEFINED global_conf_file AND NOT DEFINED CONF_FILE)
```

Comment:
I get the idea. If `global_conf_file` is set without `CONF_FILE` being defined, then only sysbuild global exists, but it's a bit hidden.

Also what this code doesn't take into consideration is the fact that `CONF_FILE` may be a list of files, in which case code below will probably fail.

Also `global_conf_file` may be an absolute path.

I wonder if an alternative could be to include `configuration_files` in a scoped context in sysbuild and thereby obtain `CONF_FILE_BUILD_TYPE` value in the common way.
If set, then that information can be passed on to extra images.